test/groovy/BuildDockerAndPublishImageStepTests.groovy

```diff
@@ -122,7 +122,7 @@ class BuildDockerAndPublishImageStepTests extends BaseTest {
   }
 
   // Return if the usual static checks had been recorded with the usual pattern
-  Boolean assertRecordIssues(String imageName = fullTestImageName) {
+  Boolean assertRecordIssues(String imageName = fullTestImageName, String platform = 'linux/amd64') {
```

Comment:
I don't see any usage of the `platform` attribute in the method `assertRecordIssues()`, is there a particular reason to add it here?

Reply:
remnants of first tests, thanks to point out,